<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication1Class</class>
 <widget class="QWidget" name="QtGuiApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication1</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>521</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="Name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="NameEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Number">
      <property name="text">
       <string>Number</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="NumberEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="Country">
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="CountryEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="School">
      <property name="text">
       <string>school</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="SchoolEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="readBtn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>390</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>read</string>
   </property>
  </widget>
  <widget class="QPushButton" name="writeBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>write</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication1.qrc"/>
 </resources>
 <connections/>
</ui>
